s3contents/genericmanager.py

```diff
@@ -292,6 +303,44 @@ def is_hidden(self, path):
         self.log.debug("S3contents.GenericManager.is_hidden '%s'", path)
         return False
 
+    post_save_hook = Any(
+        None,
+        config=True,
+        allow_none=True,
+        help="""Python callable or importstring thereof
```

Comment:
We may want to add some colour here to explain the implications of the post-save hook operating on data in object storage instead of the local fs, but I think it makes sense to keep the API the same as the original implementation (even retaining the `os_path` arg. Thoughts?

Reply:
tl;dr: `os_path` -> `path`

I think I'd rather be explicit with the implications and call it `s3_path` instead of `os_path`. I usually am in agreement and try to keep the APIs the same, but in this case I think it makes a better user experience by changing the var name. 

I'm not sold on the generic use of `path` in many of the other functions in this library, but I think it's probably better to be consistent and use `path` instead of `s3_path`. I'd probably want to refactor things to be `s3_path` just to make it explicit, but for now, I think we stick with `path` to be consistent with the rest of the lib? I could be convinced either way.

Reply:
The more I think about it the more I'm okay with just refactoring it to be `s3_path`, if we provide examples in the docs it should be clear enough. IMO It's a logical step to go from `pre:path -> post:os_path` to `pre:path -> post:s3_path`. Not to mention you'd need to rewrite your `FileSystemContentsManager` post save hook anyways so it may be more confusing to make `os_path` just `path`.